velox/functions/sparksql/JsonFunctions.h

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include "velox/functions/prestosql/json/SIMDJsonUtil.h"
+#include "velox/functions/prestosql/types/JsonType.h"
+
+namespace facebook::velox::functions::sparksql {
+
+template <typename T>
+struct JsonArrayLengthFunction {
```

Comment:
It looks like this implementation is the same with that of Presto at https://github.com/facebookincubator/velox/blob/main/velox/functions/prestosql/JsonFunctions.h#L124-L148. We can just reuse it with a registration under Spark like https://github.com/facebookincubator/velox/commit/4661b614767c1e8f7510dfcd280f0713209f10ae.